pom.xml

```diff
@@ -70,17 +98,63 @@
         </dependency>
         
     </dependencies>
+    
+    <repositories>
```

Comment:
This is already In your parent pom . I believe it will be inherited.

Reply:
Fair point, this was added from the changes i made to a previous project but i've removed it now